src/integration/integration_suite_test.go

```diff
@@ -278,6 +281,50 @@ func getSchedules(appId string) (*http.Response, error) {
 	return httpClient.Do(req)
 }
 
+func createSchedule(appId string, schedule string) (*http.Response, error) {
+	req, err := http.NewRequest("PUT", fmt.Sprintf("https://127.0.0.1:%d/v2/schedules/%s", components.Ports[Scheduler], appId), bytes.NewReader([]byte(schedule)))
+	if err != nil {
+		panic(err)
+	}
+	Expect(err).NotTo(HaveOccurred())
+	req.Header.Set("Content-Type", "application/json")
+	return httpClient.Do(req)
+}
+
+func deleteSchedule(appId string) (*http.Response, error) {
+	req, err := http.NewRequest("DELETE", fmt.Sprintf("https://127.0.0.1:%d/v2/schedules/%s", components.Ports[Scheduler], appId), strings.NewReader(""))
+	Expect(err).NotTo(HaveOccurred())
+	req.Header.Set("Content-Type", "application/json")
+	return httpClient.Do(req)
+}
+
+func getAppIdOfActiveSchedules() []string {
+	var appIds []string
+	var appId string
+	rows, e := dbHelper.Query("SELECT appid FROM activeschedule")
+	if e != nil {
+		Fail("can not get records in table activeschedule: " + e.Error())
+	}
+	defer rows.Close()
+	for rows.Next() {
+		err := rows.Scan(&appId)
+		if err != nil {
+			Fail("Fail")
+		}
+		appIds = append(appIds, appId)
+	}
+	return appIds
+}
+
+func getNumberOfActiveSchedules(appId string) int {
```

Comment:
We will use scaling engine API for  checking active schedule but still need it for test cases where in scaling engine is down.